client-misk-hibernate/src/test/kotlin/app/cash/backfila/client/misk/ClientMiskTestingModule.kt

```diff
@@ -51,19 +52,17 @@ internal class ClientMiskTestingModule(
     install(LogCollectorModule())
     install(MiskTestingServiceModule())
     install(EmbeddedBackfilaModule())
-    install(
-        BackfilaModule(
-            BackfilaClientConfig(
-                url = "test.url", slack_channel = "#test"
-            )
+    install(BackfillModule(
+        BackfilaClientConfig(
+            url = "test.url", slack_channel = "#test"
         )
-    )
+    ))
 
     bind(BackfilaClientLoggingSetupProvider::class.java)
         .to(BackfilaClientNoLoggingSetupProvider::class.java)
 
-    install(HibernateBackfillInstallModule.create<SinglePartitionHibernateTestBackfill>())
-    install(HibernateBackfillInstallModule.create<ChickenToBeefBackfill>())
-    install(HibernateBackfillInstallModule.create<RecordNoParametersConfigValuesBackfill>())
+    install(create<SinglePartitionHibernateTestBackfill>())
```

Comment:
not sure I like the readability of this

Reply:
Which part? The install of the create or the fact that it's reified?

Reply:
I preferred it without the static import, reading `install(create<>())` is a little odd but I guess it's totally fine just in this test module